src/fetch.ts

```diff
@@ -0,0 +1,42 @@
+import { info } from '@actions/core'
+
+export async function wait (delay: number): Promise<string> {
+  return await new Promise((resolve) => setTimeout(resolve, delay))
+}
+
+export async function fetchRetry (url: string, tries = 5, baseDelayMS = 1000): Promise<Response> {
+  let attempt = 0
+
+  while (tries > 0) {
```

Comment:
Do not modify `tries` and compare with `attempt`s.
```suggestion
  while (tries > attempt) {
```

Reply:
Done